topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -170,140 +157,304 @@ To estimate sequence quality along all sequences, we now use [FastQC](https://ww
 
-On the x-axis are the base position in the read, and on the y-axis quality scores. In this example, the sample contains reads that are 37 bp long. For each position, there is a box plotted to illustrate the distribution of values (with the whiskers indicating the 90th and 10th percentile scores). The blue line represents the mean score at the postion. The plot background is color-coded to identify good (green), acceptable (yellow), and bad (red) quality scores.
+On the x-axis are the base position in the read. In this example, the sample contains reads that are 37 bp long.
 
-> ### {% icon question %} Questions
+> ### {% icon comment %} Non uniform x-axis
 >
-> 1. Which Phred encoding is used in the FASTQ file for these sequences?
-> 2. How is the mean score changing along the sequence?
-> 3. Is this tendency seen in all sequences?
+> The x-axis is not uniform. It starts out with individual 1-10 bases. After that, bases are binned across a window a certain number of bases wide. Data binning means grouping and is a data pre-processing technique used to reduce the effects of minor observation errors. The number of base positions binned together depends on the length of the read. With 150 bp reads, the latter part of the plot will report aggregate statistics for 5bp windows. Shorter reads will have smaller windows and longer reads larger windows. 
 >
-> > ### {% icon solution %} Solution
-> > 1. The Phred scores are encoded using `Sanger / Illumina 1.9` (`Encoding` in the top table)
-> > 2. The mean score over the sequence is dropping at the end of the sequences. It is usual: the sequencers are incorporating more errored nucleotides at the end. But the score stays good: over 28.
-> > 3. The box plots are getting wider at the end of the sequences. It means a lot of sequences have their score dropping at the end of the sequence. At after 31 nucleotides, more than 10% of the sequences have scores below 20.
-> >
-> {: .solution }
-{: .question}
+> Binning can be removed when running FastQC
+{: .comment}
 
-FastQC produces other diagnostic plots to assess sample quality.
+For each position, a boxplot is drawn with:
+
+- the median value, represented by the central red line
+- the inter-quartile range (25-75%), represented by the yellow box
+- the 10% and 90% values in the upper and lower whiskers
+- the mean quality, represented by the blue line
+
+The y-axis shows the quality scores. The higher the score, the better the base call. The background of the  graph divides the y-axis into very good quality scores (green), scores of reasonable quality (orange), and reads of poor quality (red).
+
+It is normal with all Illumina sequencers for the median quality score to start out lower over the first 5-7 bases and to then rise. The quality of reads on most platforms will drop at the end of the read. This is often due to signal decay or phasing during the sequencing run:
+
+- Signal decay
```

Comment:
explanation of signal decay and phasing maybe in a details box?

Reply:
yeah maybe better